packages/composite-checkout/README.md

```diff
@@ -0,0 +1,559 @@
+# WP Checkout
+
+A set of React components, custom Hooks, and helper functions that together can be used to create a purchase and checkout flow.
+
+## 💰 Installation
```

Comment:
I'd keep emojis out from titles, but I'm just little boring... :-D

In some systems these titles will form automatic # links. With emojis those will look funky.